docs/src/components/Layout/GlobalNav/GlobalNav.tsx

```diff
@@ -23,17 +23,17 @@ export interface NavMenuItem {
 export interface NavProps {
   leftLinks: NavMenuItem[];
   rightLinks: NavMenuItem[];
+
+  socialLinks: NavMenuItem[];
   currentSite: string;
-  secondaryNavDesktop?: JSX.Element;
-  secondaryNavMobile?: JSX.Element;
 }
 
 export function GlobalNav({
   currentSite,
   leftLinks,
   rightLinks,
-  secondaryNavDesktop,
-  secondaryNavMobile,
+
+  socialLinks,
 }: NavProps) {
   const themeableSites: any = {
```

Comment:
Can we make the type more specific than `any`? Does this need to be an object or can it just be a boolean?